packages/table/test/quadrants/tableQuadrantTests.tsx

```diff
@@ -144,19 +144,19 @@ describe("TableQuadrant", () => {
                 const menuRenderer = sinon.stub().returns(<div className={MENU_CLASS} />);
                 const component = mountTableQuadrant({ menuRenderer });
                 expect(menuRenderer.called).to.be.true;
-                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER} > .${MENU_CLASS}`).length).to.equal(1);
+                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER} > .${MENU_CLASS}`)).to.have.lengthOf(1);
             });
 
             it("does not render menu if menuRenderer not provided", () => {
                 const component = mountTableQuadrant();
-                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children().length).to.equal(0);
+                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children()).to.have.lengthOf(0);
```

Comment:
**nit:** `.to.be.empty` seems a bit more appropriate for this type of check:
```suggestion
                expect(component.find(`.${Classes.TABLE_TOP_CONTAINER}`).children()).to.be.empty;
```

Reply:
Sure, done!